### PR TITLE
feat(ml): show custom properties for MLFeatureTable in UI

### PR DESCRIPTION
**Pull request by maaaikoool**
Adds the properties tab to MLFeatureTable, which is quite handy to add custom metadata that does not fit with the actual model.

image

## Checklist
- [X] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [X] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/mlmodel/mappers/MLFeatureTablePropertiesMapper.java [hunk 1]
```diff
@@ -41,6 +42,8 @@ public MLFeatureTableProperties apply(@NonNull final com.linkedin.ml.metadata.ML
                 .collect(Collectors.toList()));
         }
 
+        result.setCustomProperties(StringMapMapper.map(mlFeatureTableProperties.getCustomProperties()));
```
Comment (gabe-lyons) [inline datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/mlmodel/mappers/MLFeatureTablePropertiesMapper.java:45]: @maaaikoool I think we need a `(if mlFeatureTableProperties. getCustomProperties() != null)` check here

Comment (maaaikoool) [reply datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/mlmodel/mappers/MLFeatureTablePropertiesMapper.java:45]: I believe that `customProperties` is non nullable ( it has a `@Nonnull` annotation )

Comment (maaaikoool) [reply datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/mlmodel/mappers/MLFeatureTablePropertiesMapper.java:45]: Yep, it is not optional https://github.com/datahub-project/datahub/blob/master/metadata-models/src/main/pegasus/com/linkedin/common/CustomProperties.pdl#L15

Comment (maaaikoool) [reply datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/mlmodel/mappers/MLFeatureTablePropertiesMapper.java]: I've double checked with other mappers and it seems that indeed we are using a conditional assignment checking for `hasCustomProperties` so I've added it 👌

**Review by gabe-lyons — APPROVED** [review]
thanks!

**Comment by maaaikoool** [comment]
thanks for the review!

**Pull request merged**